PowerPointLabs/PowerPointLabs/PositionsLab/PositionsPaneWPF.xaml.cs

```diff
@@ -1099,8 +1099,42 @@ public void ExecutePositionsAction(Action<ShapeRange> positionsAction, bool isPr
             {
                 if (simulatedShapes != null)
                 {
-                    simulatedShapes.Delete();
-                    GC.Collect();
+                    try
+                    {
+                        simulatedShapes.Delete();
+                        GC.Collect();
+                    }
+                    // Catch corrupted shapes
+                    catch (System.Runtime.InteropServices.COMException)
+                    {
+                        // Remove all simulated shapes manually
+                        for (int i = 0; i < simulatedShapes.Count; i++)
+                        {
+                            // This method to remove duplicated shapes might fail for non-corrupted shapes when mixing good/bad shapes
+                            if (this.GetCurrentSlide().Shapes[1].Name.Contains("_Copy"))
+                            {
+                                this.GetCurrentSlide().Shapes[1].Delete();
+                            }
+                        }
+                        /*
```

Comment:
Seems like the codestyle we follow recommend using /// or // rather than /* */.
https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-CSharp.adoc